src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverProjectExtension.kt

```diff
@@ -132,6 +138,18 @@ public interface KoverProjectExtension {
     }
 }
 
+/**
+ * Configurations of default Kover artifact.
+ */
+public interface DefaultArtifactConfigs {
```

Comment:
The documentation is really unclear here for one who has no understanding of Kover internals; probably worth revisiting it